website/docs/cloud-docs/api-docs/policy-sets.mdx

```diff
@@ -40,7 +40,7 @@ description: >-
 
 [Policy Enforcement](/terraform/cloud-docs/policy-enforcement) lets you use the policy-as-code frameworks Sentinel and Open Policy Agent (OPA) to apply policy checks to Terraform Cloud workspaces.
 
-[Policy sets](/terraform/cloud-docs/policy-enforcement/manage-policy-sets) are collections of policies that you can apply globally or to specific [projects](/terraform/cloud-docs/workspaces/organize-workspaces-with-projects) and workspaces. For each run in the selected workspaces, Terraform Cloud checks the Terraform plan against the policy set.
+[Policy sets](/terraform/cloud-docs/policy-enforcement/manage-policy-sets) are collections of policies that you can apply globally or to specific [projects](/terraform/cloud-docs/workspaces/organize-workspaces-with-projects) and workspaces, except the ones in the exclusions list. For each run in the selected workspaces, Terraform Cloud checks the Terraform plan against the policy set.
```

Comment:
I agree with Keran, I don't think we should mention exclusions here because we are still explaining what a policy set is. We want folks to have that firm understanding before building on that knowledge:
```suggestion
[Policy sets](/terraform/cloud-docs/policy-enforcement/manage-policy-sets) are collections of policies that you can apply globally or to specific [projects](/terraform/cloud-docs/workspaces/organize-workspaces-with-projects) and workspaces. For each run in the selected workspaces, Terraform Cloud checks the Terraform plan against the policy set.
```